### DISCUSSION TITLE
Illegal instruction (core dumped) :: Gatsby v4 :: AMD Phenom x6 1075t

### DISCUSSION THREAD

**Discussion by eddi00** [gatsbyjs/gatsby#36197] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Hi there!

When I create a new Gatsby project with the command `gatsby new project` and then run `gatsby develop`, I get an instruction error. I think the problem relies in gatsby-plugin-sharp and gatsby-transformer-sharp, commenting the plugins in gatsby-config.js helps.

Version 3 of this plugins doesn't give me the same error (tried compiling and running this project [https://github.com/john-smilga/[gatsby-v3-tutorial-recipes]](https://github.com/john-smilga/gatsby-v3-tutorial-recipes) )

### Reproduction Link

https://github.com/eddi00/gatsby-bug-plugin-image

### Steps to Reproduce

1. Run `gatsby new project`
2. Run `gatsby develop`
3. The same happens with a minimal gatsby setup with gatsby-plugin-image and co. installed.

Output: 
`$ gatsby develop`
success compile gatsby files - 1.091s
success load gatsby config - 0.070s
⠋ load plugins
Illegal instruction (core dumped)

### Expected Result

Project compiled and served.

### Actual Result

success compile gatsby files - 1.091s
success load gatsby config - 0.070s
⠋ load plugins
Illegal instruction (core dumped)

### Environment

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04 LTS 22.04 LTS (Jammy Jellyfish)
    CPU: (6) x64 AMD Phenom(tm) II X6 1075T Processor
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 14.19.2 - ~/.nvm/versions/node/v14.19.2/bin/node
    Yarn: 1.22.19 - /snap/bin/yarn
    npm: 8.10.0 - ~/.nvm/versions/node/v14.19.2/bin/npm
  Browsers:
    Firefox: 100.0.2
  npmPackages:
    gatsby: ^4.14.1 => 4.14.1
    gatsby-plugin-gatsby-cloud: ^4.14.0 => 4.14.0
    gatsby-plugin-image: ^2.14.1 => 2.14.1
    gatsby-plugin-manifest: ^4.14.0 => 4.14.0
    gatsby-plugin-offline: ^5.14.1 => 5.14.1
    gatsby-plugin-react-helmet: ^5.14.0 => 5.14.0
    gatsby-plugin-sharp: ^4.14.1 => 4.14.1
    gatsby-source-filesystem: ^4.14.0 => 4.14.0
    gatsby-transformer-sharp: ^4.14.0 => 4.14.0
```

**Comment by LekoArts** [comment]
In this case I'd suggest opening an issue at https://github.com/lovell/sharp and using https://www.npmjs.com/package/segfault-handler to provide more details. Could be that your CPU is not supported anymore

**Reply by developer8622** [reply]
I am also getting the same error after run command "npm run develop" and "gatsby develop"
OUTPUT

success compile gatsby files - 1.091s
success load gatsby config - 0.070s
⠋ load plugins
Illegal instruction (core dumped)

**Comment by andrekutianski** [comment]
same problem here
